template.yaml

```diff
@@ -265,6 +265,32 @@ Resources:
             Method: get
             RestApiId: !Ref NvaSearchApi
 
+  SearchAwsResourcesFunction:
```

Comment:
Does this need a mapping in OpenApi spec?

Reply:
`ResourcePagedSearchHandlerSws` is the one that has to go, but I wanted to wait until  `ResourcePagedSearchHandlerAws` was deployed, befroe removing it.